src/pages/ChatFinderPage/index.tsx

```diff
@@ -162,12 +164,13 @@ function SearchPage({betas, isSearchingForReports, navigation}: SearchPageProps)
     return (
         <ScreenWrapper
             includeSafeAreaPaddingBottom={false}
-            testID={SearchPage.displayName}
+            testID={ChatFinderPage.displayName}
             onEntryTransitionEnd={handleScreenTransitionEnd}
             shouldEnableMaxHeight
             navigation={navigation}
         >
             <HeaderWithBackButton
+                // ASK: What text do we want to use for the title?
```

Comment:
@Expensify/marketing could you take a look at this one? We're introducing a new search page, so the current search page will need to be renamed. I suggested to use `Find` here, but would love your opinion. For reference, this is the page

![Screenshot 2024-04-15 at 10 06 11 AM](https://github.com/Expensify/App/assets/22219519/5a542893-c493-4213-8420-c3f80f8cdafd)

Reply:
Once we resolve that, I'll record videos for this PR :)

Reply:
@luacmartins can you actually bring this question out into #marketing? This seems like a slightly larger change that we should discuss and align on first in Slack. Thanks!